src/app/components/task/EditTaskDialog.js

```diff
@@ -238,13 +238,13 @@ class EditTaskDialog extends React.Component {
         ))}
         <div style={{ marginTop: units(1) }}>
           <Button onClick={this.handleAddValue.bind(this)}>
-            {this.props.intl.formatMessage(messages.addValue)}
+            {<FormattedMessage {...messages.addValue} />}
```

Comment:
Please remove outer curly braces as they're unnecessary here.
Also please add `id` and `defaultMessage` as props to the `FormattedMessage` component directly and remove the `addValue` key from `messages` object.